vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj

```
@@ -136,6 +138,7 @@
         PROJECT_ROOT_NAMESPACE=Microsoft::ReactNative;
         %(PreprocessorDefinitions)
       </PreprocessorDefinitions>
+      <PreprocessorDefinitions Condition="'$(RNW_FASTBUILD)' == 'true'">RNW_FASTBUILD;%(PreprocessorDefinitions)</PreprocessorDefinitions>
```

```suggestion
      <PreprocessorDefinitions Condition="'$(RnwFastBuild)' == 'true'">RNW_FASTBUILD;%(PreprocessorDefinitions)</PreprocessorDefinitions>
```

nit: MSBuild property name casing